code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -18,8 +18,8 @@
 	return (armorval/max(organnum, 1))
 
 
-/mob/living/carbon/human/proc/checkarmor(obj/item/bodypart/def_zone, type)
-	if(!type)
+/mob/living/carbon/human/proc/checkarmor(obj/item/bodypart/def_zone, d_type)
+	if(!d_type)

```

Comment:
changed the argument name, we don't want args sharing names with actual variables, just  to be safe.